prac_06/guitars.py

```diff
@@ -0,0 +1,48 @@
+"""
+CP1404 Practical 6
+Guitars program
+Estimated time: 20 minutes
+Actual time:
+"""
+
+from prac_06.guitar import Guitar
+
+
+def main():
+    """Program to store guitar data using classes"""
+    my_guitars = []
+
+    print("My guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = get_valid_number("Year: ")
+        cost = get_valid_number("Cost: ")
+        guitar_data = Guitar(name, year, cost)
+        my_guitars.append(guitar_data)
+        print(guitar_data, "added.")
+        name = input("Name: ")
+    print("These are my guitars:")
+    vintage_string = ""
+    for i, guitar in enumerate(my_guitars, 1):
+        vintage_string = determine_if_vintage(guitar)
+        print("Guitar {0}: {1.name:>20} ({1.year}), worth ${1.cost:10,.2f}{2}".format(i, guitar, vintage_string))
+
+
+def get_valid_number(prompt):
+    """Get a valid number from user."""
+    number = float(input(prompt))
+    while number < 1:
+        print("Invalid number")
+        number = float(input(prompt))
+    return number
+
+
+def determine_if_vintage(guitar):
```

Comment:
Does the function in the class (is_vintage()) already do this? Try add it to the for loop.

Reply:
I'm unsure what you mean by this. The "is_vintage" function only checks if it is greater than the vintage age. Do you mean the for loop that enumerates through the guitars list? Can you please explain a little more? Thanks :))

Reply:
You don't need a function just to call a method.
`guitar.is_vintage()` does the same thing as `determine_if_vintage(guitar)`